src/views/Home/components/FAQ/faq.scss

```diff
@@ -1,3 +1,106 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
+@import url("https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap");
 
-/* Delete these lines when you begin development */
+.FAQ {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  z-index: 0;
+  height: 1000px;
+
+  &__whiteRect {
+    position: absolute;
+    width: 80%;
+    height: 80%;
+    background: var(--white__off);
+    box-shadow: 0 0 20px 11px rgba(89, 93, 130, 0.5);
+    border-radius: 30px;
+    z-index: 1;
+  }
+
+  &__pinkRect {
+    position: absolute;
+    width: 100%;
+    bottom: 0;
+    height: 85%;
+    background: var(--pink__light);
+    border-radius: 0 0 30px 30px;
+    z-index: 2;
+  }
+
+  &__text--header {
+    @include typography.title-text;
+
+    position: absolute;
+    bottom: calc(85% + 0.5em);
+    left: 1em;
+    color: var(--pink);
+
+    @include responsiveness.respond(tab-port) {
+      bottom: 85%;
+    }
+  }
+
+  &__text--question {
+    @include typography.source-description-purple;
+
+    font-style: normal;
+    font-weight: bold;
+    color: var(--pink);
+    margin: 1em;
+  }
+
+  &__text--answer {
```

Comment:
add padding here to alleviate the hard cutoff here & add some negative space at the base of the component to frame the text better
![Screenshot from 2020-12-25 18-55-46](https://user-images.githubusercontent.com/19658882/103144808-be0ddf00-46e4-11eb-8cf4-ed1958396382.png)